src/cps/loader.cpp

```diff
@@ -7,19 +7,22 @@
 #include "cps/error.hpp"
 #include "cps/utils.hpp"
 
+#include <algorithm>
```

Comment:
NBD, but it's nice to keep the includes for each dependency grouped together. It's also nice to sort them from high level to low level with the in-project headers at the top and standard libraries at the bottom.